### PR TITLE
01/05/2024 [FEAT]: Add notification page

### PR DESCRIPTION
**Pull request by habeeb51YA**
what was done: 
Add Notification page

### CODE REVIEW

**Comment by as-abdulsuleiman** [comment]
@habeeb51YA Pull before making any changes. The Notification batches are missing. For Flagged Posts both the flag and the red dot are missing and also make sure to add the red dot to others

 

Screenshot 2024-05-03 at 10 13 33 AM

File: src/app/(admin)/notifications/page.tsx [hunk 1]
```diff
@@ -1,11 +1,556 @@
...
+                  </div>
+                  <div className="mt-0 text-[12px] py-0">
+                    {formatDistanceToNow(new Date(value?.createdAt), {
+                      addSuffix: true,
+                    })}
+                  </div>
+                </div>
+                <div className="mt-2 ml-auto mr-0">
+                  {" "}
+                  {/* <Link
```
Comment (as-abdulsuleiman) [inline src/app/(admin)/notifications/page.tsx:473]: @habeeb51YA why do we still have this

**Comment by as-abdulsuleiman** [comment]
@habeeb51YA work on the comment above

**Review by as-abdulsuleiman — CHANGES_REQUESTED** [review]

File: src/app/(admin)/notifications/page.tsx (hunk 1 above)
Comment (habeeb51YA) [reply src/app/(admin)/notifications/page.tsx:473]: I left it just incase in the future we want to add Coach Verification detail page

**Review by as-abdulsuleiman — APPROVED** [review]

**Pull request merged**